build/depends.py

```diff
@@ -1227,6 +1227,7 @@ def sources(self, build):
                    "src/track/cue.cpp",
                    "src/track/cueinfo.cpp",
                    "src/track/cueinfoimporter.cpp",
+                   "src/track/frame.cpp",
```

Comment:
@hacksdump All operations on `Frame` should be inlined and defined in the header file. No need for a .cpp file.

Reply:
Please edit this commit directly and do a force push. No need to keep the previous changes around.